packages/kbn-dev-utils/src/precommit_hook/cli.ts

```diff
@@ -24,14 +24,21 @@ import { REPO_ROOT } from '@kbn/utils';
 
 import { run } from '../run';
 import { SCRIPT_SOURCE } from './script_source';
-import { getGitDir } from './get_git_dir';
+import { getGitDir, isCorrectGitVersionInstalled } from './git_utils';
 
 const chmodAsync = promisify(chmod);
 const writeFileAsync = promisify(writeFile);
 
 run(
   async ({ log }) => {
     try {
+      if (!(await isCorrectGitVersionInstalled())) {
+        log.warning(
```

Comment:
Do we plan on making this a failing error once installing the precommit hook is optional?

Reply:
Good catch. I've changed it to throw an error instead and I'll merge this PR after https://github.com/elastic/kibana/pull/83566